src/commandExecution.ts

```diff
@@ -97,6 +99,9 @@ export class CommandExecution extends AsyncCreatable {
       devhubId: this.devhubId,
       orgApiVersion: this.orgApiVersion,
       devhubApiVersion: this.devhubApiVersion,
+      specifiedEnvs: envs.specifiedEnvs.join(' '),
```

Comment:
I'm not super familiar with AppInsights but is this the most usable data structure? I'd think that an array would be more usable... but maybe app insights doesn't do arrays

Reply:
telemetry-lib wants 

```
export type Attributes = {
    [key: string]: string | number | boolean | null | undefined;
```

I've never really questioned it